INSTALL

```diff
@@ -55,34 +55,40 @@ If you are installing dependencies with a package manager on Linux,
 you may need to install the development packages (look for a "-dev"
 postfix) in addition to the core packages.
 
-python 2.7 or later (http://www.python.org/)
-    cf-units requires Python 2.7 or later, but is not currently compatible with
-    Python 3.
+python 2.7 or later (https://www.python.org/)
+    cf-units requires Python 2.7 or later.
```

Comment:
This line feels tautological now – it's not saying anything that hasn't already been said on the line above. If we wanted to be explicit and say something like `cf-units is compatible with Python 2 and Python 3`, but, really, that's also a given...